pkg/types/types.go

```diff
@@ -56,6 +56,7 @@ Network controller subclasses. ref: https://pci-ids.ucw.cz/read/PD/02
 07	Infiniband controller
 08	Fabric controller
 80	Network controller
+12	Processing Accelerator
```

Comment:
This list above is about network subclasses, to make it cosistent, maybe we should either of below:
1) list subclasses for `Processing Accelerator`
2) list the only `PCI Device Classes` for `Network controller` and `Processing Accelerator`

Reply:
Can you please take another look please?